components/x-podcast-launchers/readme.md

```diff
@@ -55,5 +55,6 @@ Feature               | Type     | Required | Notes
 `acastRSSHost`        | String   | Yes      | e.g. 'https://acast.access.com'
 `conceptId`           | String   | Yes      |
 `renderFollowButton`  | Function | No       | Optional render prop for the follow button
+`isApp`               | Boolean  | No       | Defaults to true. Currently only used to hide podcast app launchers on wide screen + desktop where the links do nothing because there is no app to launch.
```

Comment:
i'd prefer a more descriptive name for this property, something like `showOnAllBreakpoints`?

Reply:
Thanks bren. I was going for an easy to understand variable eg is this the app - yes or no? which one could then hang behaviours off of - in this case showing or hiding the podcast launcher buttons. 
I prefer your name and semantic change though - it's simpler. I'll implement.